src/test/scala/ai/privado/languageEngine/kotlin/tagger/collection/KotlinCollectionTaggerTest.scala

```diff
@@ -0,0 +1,59 @@
+package ai.privado.languageEngine.kotlin.tagger.collection
+
+import ai.privado.cache.RuleCache
+import ai.privado.model.*
+import io.shiftleft.semanticcpg.language.*
+
+class KotlinCollectionTaggerTest extends KotlinTaggingTestBase {
+  private val collectionRule = List(
+    RuleInfo(
+      "Collections.Spark.HttpFramework",
+      "Spark Java Http Framework Endpoints",
+      "",
+      FilterProperty.METHOD_FULL_NAME,
+      Array(),
+      List(".*.get.*"),
+      false,
+      "",
+      Map(),
+      NodeType.REGULAR,
+      "",
+      CatLevelOne.COLLECTIONS,
+      catLevelTwo = Constants.default,
+      Language.KOTLIN,
+      Array()
+    )
+  )
+
+  "Simple Spark Http sample" should {
+    val ruleCache: RuleCache = getRuleCache
+    val fileContents = """
+        |import spark.*;
+        |
+        |fun main(args: Array<String>) {
+        |    Spark.get("/hello") { req, res ->
```

Comment:
Have only written one test. Should more be written to show other cases or is this sufficient ?

Reply:
If you have more unit test cases around this scenario, those are welcome

Reply:
Added more unit tests for Kotlin and Java.